lib/libfloat.dl

```diff
@@ -0,0 +1,86 @@
+// Library with operations on floating point values (float and double)
+// All float operations have suffix _f, and all double operations have suffix _d
+
+extern function floor_f(f: float): float
```

Comment:
No tests for any of these functions? There's a script for testing libraries now (see `test/datalog_tests/test-libs.sh`)